### PR TITLE
uKBlvQDD: Enable some ignored tests

### PR DESCRIPTION
**Pull request by loop-evgeny**

### CODE REVIEW

File: Loop54.Tests.Shared/CallMethods.cs [hunk 1]
```diff
@@ -67,23 +67,21 @@ public void GetRelatedEntitiesHasResults()
...
 
         [Test]
-        [Ignore("Not released to HelloWorld engine yet")]
         public void GetComplementaryEntitiesHasResults()
         {
             //Should be a wheat flour
             var response = GetClient().GetComplementaryEntities(WrapRequest(new GetComplementaryEntitiesRequest("Product", "13")));
-            Assert.Greater(response.Results.Count, 0);
-            Assert.Greater(response.Results.Items.Count, 0);
+            Assert.AreEqual(response.Results.Count, 0); // Currently no complementary entities in HelloWorld engine
```
Comment (loop-evgeny) [inline Loop54.Tests.Shared/CallMethods.cs:74]: Not great, but at least it checks that the API call does not fail - better than nothing

File: Loop54.Tests.Shared/CallMethods.cs [hunk 2]
```diff
@@ -113,7 +111,6 @@ public void GetEntitiesByAttributeSingleValueHasResults()
         }
 
         [Test]
-        [Ignore("Not released to HelloWorld engine yet")]
```
Comment (loop-evgeny) [inline Loop54.Tests.Shared/CallMethods.cs:116]: This one works

File: Loop54.Tests.Shared/Loop54Client.cs [hunk 3]
```diff
@@ -79,27 +77,35 @@ public void SearchWithMetaDataOverrides()
         }
 
         [Test]
-        public void SearchWithMetaDataAndClientInfoOverrides() => CallWithOverrides((c, r) => c.Search(r), new SearchRequest("pork belly"), "search");
+        public void SearchWithMetaDataAndClientInfoOverrides() =>
```
Comment (loop-evgeny) [inline Loop54.Tests.Shared/Loop54Client.cs:80]: Just reformatting here to avoid lines > 150 characters - no real changes

**Review by oleg-loop54 — APPROVED** [review]
LGTM!

**Pull request merged**